Services/UI/OJS.Services.Ui.Models/Contests/ContestServiceModel.cs

```diff
@@ -190,7 +175,7 @@ public bool ResultsArePubliclyVisible
             .ForMember(
                 d => d.ProblemsCount,
                 opt => opt.MapFrom(s => s.ProblemGroups.SelectMany(pg => pg.Problems).Count(p => !p.IsDeleted)))
-            .ForMember(d => d.ContestType, opt => opt.MapFrom(s => s.Type))
+            .ForMember(d => d.Type, opt => opt.MapFrom(s => s.Type))
```

Comment:
The type of the property in both the destination and the source is ContestType and the names are identical. Don't the properties get mapped automatically without having to explicitly specify it here ?